networks/rpc/service.go

```diff
@@ -285,7 +285,7 @@ func isSubscriptionType(t reflect.Type) bool {
 	return t == subscriptionType
 }
 
-// isPubSub tests whether the given method has as as first argument a context.Context and
+// isPubSub tests whether the given method has as its first argument a context.Context and
```

Comment:
```suggestion
// isPubSub tests whether the given method's first argument is a context.Context and
```

see https://github.com/ethereum/go-ethereum/blob/master/rpc/service.go#L230

Reply:
Thanks for your suggestion. Modified.